service/gcs/stdio/stdio.go

```diff
@@ -4,10 +4,15 @@ import (
 	"io"
 	"os"
 	"sync"
+	"sync/atomic"
+	"syscall"
+	"unsafe"
 
 	"github.com/Microsoft/opengcs/service/gcs/transport"
 	"github.com/Sirupsen/logrus"
 	"github.com/pkg/errors"
+
+	"golang.org/x/sys/unix"
```

Comment:
Nit: To keep with the pattern we've started following (using goimports to perform formatting), this import should probably go with the above github.com imports.